configure.py

```diff
@@ -791,10 +793,13 @@ def __init__(self, infofile):
         self.lives_in = dirname
         if basename == 'info.txt':
             (obj_dir, self.basename) = os.path.split(dirname)
-            if os.access(os.path.join(obj_dir, 'info.txt'), os.R_OK):
-                self.parent_module = os.path.basename(obj_dir)
-            else:
-                self.parent_module = None
+            self.parent_module = None
+
+            while obj_dir:
+                if os.access(os.path.join(obj_dir, 'info.txt'), os.R_OK):
+                    self.parent_module = os.path.basename(obj_dir)
+                    break
+                (obj_dir, _) = os.path.split(obj_dir)
```

Comment:
This fixes an issue where parent modules could not be found when modules where nested into intermediate directories that did not contain an `info.txt`. One might argue that this is not strictly required anymore, given that such directories are likely going to be `Virtual` modules now. Nevertheless, this isn't enforced so probably it does not hurt to keep this fix.